src/app/pages/wallets/my-wallets/my-wallet/pages-wallets-my-wallets-my-wallet.component.ts

```diff
@@ -8,4 +8,10 @@ import { MyWallet } from '../pages-wallets-my-wallet.model';
 })
 export class PagesWalletsMyWalletsMyWalletComponent {
   @Input() public wallet!: MyWallet;
-}
+
+  @Output('walletEdit') public walletEdit = new EventEmitter<MyWallet>();
```

Comment:
There is no reason to use `'walletEdit'` here. You can just go with `@Output()`. You should pass string to this when you want to have different name in template than in the class.
For example you can have:
```typescript
@Output('otherWalledEdit') public walletEdit
```
then in the html you have:
```html
<app-my-wallet (otherWalletEdit)=""></app-my-wallet>
```
and in this class you still use it like this:
```typescript
this.walletEdit.emit()
```